gocomplete/complete.go

```diff
@@ -15,6 +22,10 @@ var (
 )
 
 func main() {
+	if os.Getenv(envVerbose) != "1" {
```

Comment:
Maybe this could be a library feature and not specifically for gocomplete?
Can this change be in https://github.com/posener/complete/blob/v1/log.go?

Reply:
Yeah, I think it does make sense for something like it to be a part of the library. I'm really not sure about the exact design though. Was there any specific reason why the v1's `Log` was dropped from v2?

Reply:
Oh, so sorry.
This `v1` default branch is confusing.
Yeh, I guess that the logs were removed because of the problem that you are facing.